src/bundler.ts

```diff
@@ -195,7 +195,13 @@ export class EsbuildBundler {
           printBuildMessages(error as BuildFailure, { prefix: 'Build ' });
         }
 
-        process.env.ESBUILD_BINARY_PATH = originalEsbuildBinaryPath;
+        /**
+         * only reset `ESBUILD_BINARY_PATH` if it was explicitly set via the construct props
+         * since `esbuild` itself sometimes sets it (eg. when running in yarn 2 plug&play)
+         */ 
```

Comment:
```suggestion
         */
```
The extra whitespace is causing the build to fail 🤦🏻